### DISCUSSION TITLE
Recreate Network Interfaces

### DISCUSSION THREAD

**Discussion by k1rb** [Security-Onion-Solutions/securityonion#8980] [General] [answered]
Hello,

tl;dr: Looking for documentation or help rebuilding the network configuration after hardware change.

I recently migrated a security onion vm to a new hypervisor, doing so changed the vm's network device addresses, from enp1s0 to enp6s18, and so on. I thought I could correct this by assigning the new device to eth0 (bond0-slave-eth0) but this didn't solve the issue and I decided to recreated all of the interfaces including bond0. I still can't reach the web interface.

The MAC address has changed on the network interfaces, the device names have changed... but the IP addresses and the networks are the same.

Thanks in advance!

**Comment by dougburks** [comment, marked as answer]
For the management interface, recovery may be different depending on whether you installed on CentOS or Ubuntu. So you should first determine your base OS and then you can search Google for specific instructions for that base OS.

For the sniffing interface, you might be able to add it to `bond0` using so-monitor-add:
https://docs.securityonion.net/en/2.3/so-monitor-add.html

If all else fails, it may be quicker and easier to simply perform a fresh installation.

**Reply by k1rb** [reply]
I was using the CentOS 7 based ISO, I was having issues with both interfaces, and after trying pretty much everything short of reverse engineering I decided to start over.

Thanks for your time!
